simulations/vip-421/ethereum.ts

```diff
@@ -0,0 +1,169 @@
+import { expect } from "chai";
+import { Contract } from "ethers";
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+import { expectEvents } from "src/utils";
+import { forking, testForkedNetworkVipCommands } from "src/vip-framework";
+
+import {
+  ETHEREUM_USDC_PRIME_CONVERTER,
+  ETHEREUM_USDT_PRIME_CONVERTER,
+  ETHEREUM_WBTC_PRIME_CONVERTER,
+  ETHEREUM_WETH_PRIME_CONVERTER,
+  ETHEREUM_XVS_VAULT_CONVERTER,
+  ethereumBaseAssets,
+  ethereumUSDCPrimeConverterTokenOuts,
+  ethereumUSDTPrimeConverterTokenOuts,
+  ethereumWBTCPrimeConverterTokenOuts,
+  ethereumWETHPrimeConverterTokenOuts,
+  ethereumXVSVaultConverterTokenOuts,
+} from "../../vips/vip-421/addresses";
+import vip421PartA from "../../vips/vip-421/bscmainnetPartA";
+import vip421PartB from "../../vips/vip-421/bscmainnetPartB";
+import CONVERTER_ABI from "./abi/Converter.json";
+
+forking(21630221, async () => {
+  let converter: Contract;
+
+  describe("Incentives before VIP", () => {
+    it("Incentives in USDT converter", async () => {
+      await Promise.all(
+        ethereumUSDTPrimeConverterTokenOuts.map(async token => {
+          converter = await ethers.getContractAt(CONVERTER_ABI, ETHEREUM_USDT_PRIME_CONVERTER);
+          const incentiveAndAccess = await converter.conversionConfigurations(ethereumBaseAssets[0], token);
+          if (token === "0xd3CC9d8f3689B83c91b7B59cAB4946B063EB894A") {
+            expect(incentiveAndAccess[0]).to.equal("0");
+          } else {
```

Comment:
We shouldn't need this edge case, because we shouldn't configure the conversion of XVS

Reply:
https://github.com/VenusProtocol/vips/commit/5b7efc549db5f4f9e6144fa61e023b5e4d3cae5c